ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/json/IsJsonString.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf.json;
+
+import io.confluent.ksql.function.FunctionCategory;
+import io.confluent.ksql.function.KsqlFunctionException;
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
+import io.confluent.ksql.util.KsqlConstants;
+
+@UdfDescription(
+    name = "IS_JSON_STRING",
+    category = FunctionCategory.JSON,
+    description = "Given a string, returns true if it can be parsed as a valid JSON value, false"
+        + " otherwise.",
+    author = KsqlConstants.CONFLUENT_AUTHOR)
+public class IsJsonString {
+
+  @Udf
+  public boolean check(@UdfParameter(description = "The input JSON string") final String input) {
+    if (input == null) {
+      return false;
+    }
+
+    try {
+      return !UdfJsonMapper.parseJson(input).isMissingNode();
```

Comment:
When is `isMissingNode` true?

Reply:
The javadoc tells that `MissingNode denotes "missing nodes" along paths that do not exist. For example, if a path via element of an array is requested for an element outside range of elements in the array; or for a non-array value`.

In practice, I was able to produce a `MissingNode` by parsing _some_ invalid JSON, e.g. `OBJECT_READER.readTree("")` returns a `MissingNode`.

Reply:
Ok, sounds like this is the quickest way to verify that we have good JSON!